packages/util/src/extendedAssets/index.ts

```diff
@@ -0,0 +1,135 @@
+import type { Api } from '../api';
+import { Asset } from '../assets/types';
+import { SoulBoundToken } from './types';
+
+export class ExtendedAssetsModule<T> {
+  constructor(private readonly root: Api<T>) {}
+
+  /**
+   * Get SBT meta info by collecting it from different states
+   * @param assetId asset ID
+   *
+   */
+  public async getSbtMetaInfo(assetId: string): Promise<SoulBoundToken> {
+    const sbtSpecificInfo = await this.root.api.query.extendedAssets.soulboundAsset(assetId);
+    const sbtCommonInfo = await this.root.api.query.assets.assetInfosV2(assetId);
+
+    const { externalUrl, issuedAt, regulatedAssets } = sbtSpecificInfo.unwrapOrDefault();
```

Comment:
unwrapOrDefault() - what's the default value here? Looks like Rust `None` Object. So, here you're trying to destructurize `None` value that will lead to the error.

Reply:
<img width="921" alt="Screenshot 2024-07-12 at 12 27 01" src="https://github.com/user-attachments/assets/80f3c3f8-c052-4197-8c67-09b8e47328a1">